public/app/features/transformers/docs/content.ts

```diff
@@ -902,25 +950,27 @@ export const transformationDocsContent: TransformationDocsContentType = {
   | 2020-07-07 11:34:20 | Temperature | 25    |
   | 2020-07-07 11:34:20 | Humidity    | 22    |
   | 2020-07-07 10:32:20 | Humidity    | 29    |
+
+  This transformation helps you tailor the visual presentation of your data to focus on the most relevant information.
   `;
     },
   },
   merge: {
     name: 'Merge',
     getHelperDocs: function () {
       return `
-  Use this transformation to combine the result from multiple queries into one single result. This is helpful when using the table panel visualization. Values that can be merged are combined into the same row. Values are mergeable if the shared fields contain the same data. For information, refer to [Table panel][].
-
-  In the example below, we have two queries returning table data. It is visualized as two separate tables before applying the transformation.
+  Use this transformation to combine the results from multiple queries into a single result, which is particularly useful when using the table panel visualization. This transformation merges values into the same row if the shared fields contain the same data.
+  
+  Here's an example illustrating the impact of the "Merge Rows" transformation on two queries returning table data:
```

Comment:
Same comment as on Limit rows; this doesn't appear to be the name of the transformation. Perhaps:
```suggestion
  Here's an example illustrating the impact of the **Merge series/tables** transformation on two queries returning table data:
```